### DISCUSSION TITLE
AWS SQS Integration

### DISCUSSION THREAD

**Discussion by BjEic** [microsoft/ALAppExtensions#16995] [Ideas] [answered]
I likes the Azure Blob Storage Integration but for my scenario I need an integration to Amazon Simple Queue Services:
https://docs.aws.amazon.com/general/latest/gr/sigv4-signed-request-examples.html#sig-v4-examples-get-auth-header

One issue is in the Codeunit 1279 GenerateHashBytes returns an empty string for an empty InputString.
Another is that the HashBytes are returned in capitals.

Edit:
An empty input String returns 
SHA256("") =
e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
Source: https://de.wikipedia.org/wiki/SHA-2

**Comment by pri-kise** [comment, marked as answer]
Releated Issue: https://github.com/microsoft/ALAppExtensions/issues/16998 was closed with a note that this will be fixed a new future release.
